### ISSUE TITLE
Question on  the trained model for Prediction

### ISSUE THREAD

**Issue by minho17** [akaraspt/deepsleepnet#19]
Hi, I am a researcher studing CNN and your code is very helpful to me, thank you!

However, I have one question.

If you give me a short comment, also it will very helpful to me.

When I reproduce the prediction accuracy, it was slightly lower (1.5 %) than that in the paper.

I think that the reason is the modification of original code by me to run the code in my computer.

Or, is any additional processing needed to reproduce  the accuracy?

For example, the code saves the trained model after pre-determined epochs and uses the saved modle

for prediction.

However, the model can be saved in a certain epoch by monitoring the validation accuracy.

Is the accuracy in the paper calculated by this or other similar process?  

Thank you for reading this.

**Comment by akaraspt** [comment]
Hi,

There is no additional processing needed. The code should be able to reproduce the results that are close to the ones reported in the paper. The accuracy may vary depending on the initialized weights at the beginning of the training.

The accuracy in the paper was the one that we got from the model trained until the final epoch (i.e., we did not choose the best one; we chose the last one). The reason behind this is to prevent the bias from choosing the model that perform well in the validation set (as we applied the k-fold cross validation and we dont have the test set).

Hope this help.
Akara
